File: WeatherApp/app/src/main/java/com/example/android/Task03/Retrofit/MainDataValues.java
package com.example.android.Task03.Retrofit;

import com.google.gson.annotations.SerializedName;

public class MainDataValues {

    @SerializedName("temp")
    double temp;

    @SerializedName("humidity")
    String humidity;

    public String getTemp() {
        int t = (int)temp;
            char grad = 176;
            return t + "" + grad;
        }

    public String getHumidity() {
        return humidity;
    }
}

File: WeatherApp Kotlin/app/src/main/java/com/example/android/weatherApp/retrofit/MainDataValues.kt
package com.example.android.weatherApp.retrofit

import com.google.gson.annotations.SerializedName

class MainDataValues {
    @SerializedName("temp")
    var temp = 0.0

    @SerializedName("humidity")
    var humidity: String? = null
    fun getTemp(): String {
        val t = temp.toInt()
        val grad = 176.toChar()
        return t.toString() + "" + grad
    }
}